packages/discord-bot/package.json

```diff
@@ -8,6 +8,9 @@
   "type": "module",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "engines": {
+    "node": ">=16.18.0"
```

Comment:
I was running the discord-bot with npm 16.13 locally and it was failing. Upgrading to v16 LTS fixed it